de.dlr.sc.virsat.server.test/src/de/dlr/sc/virsat/server/resources/AccessTestResourceTest.java

```diff
@@ -0,0 +1,73 @@
+/*******************************************************************************
+ * Copyright (c) 2008-2019 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package de.dlr.sc.virsat.server.resources;
+
+import static org.junit.Assert.assertEquals;
+
+import java.net.URI;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.UriBuilder;
+
+import org.glassfish.jersey.client.ClientConfig;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import de.dlr.sc.virsat.server.test.AGitAndJettyServerTest;
+
+public class AccessTestResourceTest extends AGitAndJettyServerTest {
+
+	@Before
+	public void setUp() throws Exception {

```

Comment:
No need to overwrite setUp/tearDown just to call their super methods.

Reply:
right.